proto/grp.proto

```diff
@@ -61,7 +72,59 @@ message GetGroupResponse {
   string owned_domain = 3;
 }
 
+message GetGroupUsersRequest {
+  context.RequestContext request_context = 1;
+
+  // The ID of the group to get the users from.
+  string group_id = 2;
+
+  // Membership status filter (optional).
+  GroupMembershipStatus group_membership_status = 3;
```

Comment:
consider making this repeated now -- this will allow you to getch multiple statuses at once?

Reply:
Seems unlikely that we'll need that since there's only 2 statuses and UNKNOWN means "all". It can be made repeated later if we do need it, without needing to deprecate this field